.tool-versions

```diff
@@ -1,2 +1,2 @@
-erlang 23.0.2
-elixir 1.10.3-otp-23
+erlang 24.0.2
```

Comment:
Usually you'll have to update a `Dockerfile` as well, when changing the `.tool-versions`. But from a quick glance, and referring to the deleted `build_push.sh` script, I think it uses `./Dockerfile` and `apps/train_loc/Dockerfile` is unused. Can you update that first one and delete the second one?

Reply:
Didn't notice that, thank you for catching it.